pandas/core/panel.py

```diff
@@ -132,6 +130,18 @@ def _constructor(self):
 
     def __init__(self, data=None, items=None, major_axis=None, minor_axis=None,
                  copy=False, dtype=None):
+        # deprecation GH13563
+        warnings.warn("\nPanel is deprecated and will be removed in a "
+                      "future version.\nThe recommended way to represent "
+                      "these types of 3-dimensional data are with a "
+                      "MultiIndex on a DataFrame, via the "
+                      "Panel.to_frame() method\n"
+                      "alternatively, you can use the `xarray package "
+                      "<http://xarray.pydata.org/en/stable/>`__.\n"
```

Comment:
you don't need to use the rst syntax with `` `..`__ `` here, as this is for console output